dashboard/app/views/levels/_form.html.haml

```diff
@@ -7,11 +7,9 @@
       = f.label :name
       = f.text_field :name, placeholder: 'Level Name'
   / Only enter name to create new records of certain level types.
-  - unless @level.new_record? && (@level.is_a?(Grid) || @level.is_a?(Calc) || @level.is_a?(Eval) || @level.is_a?(Artist) || @level.is_a?(Applab) || @level.is_a?(NetSim))
-    / only Blockly-type levels have Concepts
-    = render partial: 'levels/editors/concepts', locals: {f: f} if @level.is_a? Blockly
-    / Video dropdown not yet implemented for dsl-defined levels
-    = render partial: 'levels/editors/video', locals: {f: f} unless @level.is_a? DSLDefined
+  - unless @level.new_record? && @level.is_a?(Blockly)
```

Comment:
Hmm what do you mean?  This is the same as the removed line 10 above, GitHub just aligned the diff in a weird way.  The following should be equivalent:
```ruby
@level.is_a?(Grid) || @level.is_a?(Calc) || @level.is_a?(Eval) || @level.is_a?(Artist) || @level.is_a?(Applab) || @level.is_a?(NetSim)
```
```ruby
@level.is_a?(Blockly)
```
I verified with:
```sh
grep '< Blockly' -R dashboard/app/models 
dashboard/app/models/applab.rb:class Applab < Blockly
dashboard/app/models/artist.rb:class Artist < Blockly
dashboard/app/models/calc.rb:class Calc < Blockly
dashboard/app/models/eval.rb:class Eval < Blockly
dashboard/app/models/grid.rb:class Grid < Blockly
dashboard/app/models/net_sim.rb:class NetSim < Blockly
```